app/components/views/HomePage/index.js

```diff
@@ -61,6 +74,23 @@ class Home extends React.Component{
     });
   }
 
+  formatBalanceData(spendableAndLockedByDay){
+    const keys = Object.keys(spendableAndLockedByDay);
+    let data = [];
+    let newData = {};
+    for(let i=0;i< keys.length; i++){
+      const date = keys[i];
+      const values = spendableAndLockedByDay[date];
+      newData = {
+        name: date,
+        locked: values.lockedTotal/100000000,
```

Comment:
Don't use the fixed `/10000000` here. Use the selector `unitDivisor`, as it already performs the correct scaling depending on the user's settings.